source/github-helpers/search-query.ts

```diff
@@ -54,6 +54,10 @@ export default class SearchQuery {
 		this.set(this.get());
 	}
 
+	static escapeValue(value: string): string {
+		return /\s|^:/.test(value) ? `"${value}"` : value;
```

Comment:
Changed is since GitHub will add quotes if it starts with a `:` or if it contains a space you can test it https://github.com/yakov116/TestR/labels

Reply:
Doesn't look accurate: https://github.com/fregante/sandbox/issues?q=is%3Aopen+sort%3Aupdated-desc+label%3A%3Aa%3A

I think it only happens for spaces, but there might be other characters as well. It might be worth creating new labels to test this out. You can do so in a test repo from the issue sidebar

Reply:
I did you want more tests?

Reply:
I mean manual tests on GitHub.com to determine what characters triggers the quotes. From what I see, currently this should be:
```suggestion
		return value.includes(' ') ? `"${value}"` : value;
```

Reply:
But what about

https://github.com/yakov116/TestR/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22t+%3A%2B1%3A%22 ??

Reply:
maybe if it has an emojie?

Reply:
Great idea lets include!

https://github.com/markdown-it/markdown-it-emoji/blob/339abf72f84a02e66fe9a4d682a7217d1f660e6a/dist/markdown-it-emoji.js#L9-L1848

Reply:
… that label has a space

Reply:
Also emojis are fine: https://github.com/fregante/sandbox/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A😃m